open-metadata-implementation/adapters/open-connectors/event-bus-connectors/open-metadata-topic-connectors/kafka-open-metadata-topic-connector/pom.xml

```diff
@@ -31,6 +31,12 @@
         <dependency>
             <groupId>org.apache.kafka</groupId>
             <artifactId>kafka-clients</artifactId>
+            <exclusions>
```

Comment:
Can you clarify the reason for this exclusion - in the past a few issues have been hidden, only to resurface later, when excluding part of a dependency. It is sometimes needed of course, and a valid approach, but we should explain/document why

Reply:
Reverted, no longer applies

Reply:
Thanks. closing